catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/CollectionDAO.java

```diff
@@ -779,12 +780,38 @@ List<List<String>> listFromByAllPrefix(
             + "fromFQN LIKE CONCAT(:fqnPrefix, '%') AND fromType = :fromType AND toType = :toType "
             + "AND relation = :relation")
     @RegisterRowMapper(ToFieldMapper.class)
-    List<List<String>> listToByPrefix(
+    List<Triple<String, String, String>> listToByPrefix(
         @Bind("fqnPrefix") String fqnPrefix,
         @Bind("fromType") String fromType,
         @Bind("toType") String toType,
         @Bind("relation") int relation);
 
+    @SqlQuery(
+        "SELECT fromFQN, toFQN, json FROM field_relationship WHERE "
+            + "fromFQN = :fqn AND fromType = :type AND toType = :otherType AND relation = :relation "
+            + "UNION "
+            + "SELECT toFQN, fromFQN, json FROM field_relationship WHERE "
+            + "toFQN = :fqn AND toType = :type AND fromType = :otherType AND relation = :relation")
+    @RegisterRowMapper(ToFieldMapper.class)
+    List<Triple<String, String, String>> listBidirectional(
+        @Bind("fqn") String fqn,
+        @Bind("type") String type,
+        @Bind("otherType") String otherType,
```

Comment:
@mosiac1 what is this otherType supposed to store? any additional information we need can be stored here https://github.com/open-metadata/OpenMetadata/blob/main/bootstrap/sql/com.mysql.cj.jdbc.Driver/v001__create_db_connection_info.sql#L30

Reply:
it's not storing any new information, it's just referring to this [column](https://github.com/open-metadata/OpenMetadata/blob/main/bootstrap/sql/com.mysql.cj.jdbc.Driver/v001__create_db_connection_info.sql#L27)

`listBidirectional` get all field relations where one side has fqn = `fqn` and type = `type` and the other has type = `otherType`